web/projects/ui/src/app/app/sidebar-host.component.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Directive,
+  Injectable,
+} from '@angular/core'
+import {
+  AbstractTuiPortalHostComponent,
+  AbstractTuiPortalService,
+  TuiDropdownPortalService,
+} from '@taiga-ui/cdk'
+
+@Injectable({ providedIn: `root` })
+export class SidebarService extends AbstractTuiPortalService {}
+
+@Directive({
+  selector: '[tuiSidebar]',
+  standalone: true,
+  providers: [
+    { provide: TuiDropdownPortalService, useExisting: SidebarService },
+  ],
+})
+export class SidebarDirective {}
+
+@Component({
+  selector: 'sidebar-host',
+  template: '<ng-container #viewContainer></ng-container>',
+  styles: [':host { position: fixed; top: 0; }'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  providers: [
+    { provide: AbstractTuiPortalService, useExisting: SidebarService },
+  ],
+})
+export class SidebarHostComponent extends AbstractTuiPortalHostComponent {}
```

Comment:
Can you explain why this approach instead of just using the [tuiSidebar](https://taiga-ui.dev/directives/sidebar) as is?

Reply:
This way it appears under the dialogs and not over them, as dropdowns/portals do by default. Since we want to open dialogs from this sidebar and have it remain open — I've made a special container for it above all content and below dialogs.